react/Tooltip/index.jsx

```diff
@@ -1,3 +1,111 @@
-import Tooltip from '@material-ui/core/Tooltip'
+import React, { useState } from 'react'
+import Proptypes from 'prop-types'
+import MUITooltip from '@material-ui/core/Tooltip'
+import { withStyles } from '@material-ui/core/styles'
 
-export default Tooltip
+const arrowGenerator = color => {
+  return {
+    '&[x-placement*="bottom"] $arrow': {
+      top: 0,
+      left: 0,
+      marginTop: '-0.95em',
+      width: '3em',
+      height: '1em',
+      '&::before': {
+        borderWidth: '0 1em 1em 1em',
+        borderColor: `transparent transparent ${color} transparent`
+      }
+    },
+    '&[x-placement*="top"] $arrow': {
+      bottom: 0,
+      left: 0,
+      marginBottom: '-0.95em',
+      width: '3em',
+      height: '1em',
+      '&::before': {
+        borderWidth: '1em 1em 0 1em',
+        borderColor: `${color} transparent transparent transparent`
+      }
+    },
+    '&[x-placement*="right"] $arrow': {
+      left: 0,
+      marginLeft: '-0.95em',
+      height: '3em',
+      width: '1em',
+      '&::before': {
+        borderWidth: '1em 1em 1em 0',
+        borderColor: `transparent ${color} transparent transparent`
+      }
+    },
+    '&[x-placement*="left"] $arrow': {
+      right: 0,
+      marginRight: '-0.95em',
+      height: '3em',
+      width: '1em',
+      '&::before': {
+        borderWidth: '1em 0 1em 1em',
+        borderColor: `transparent transparent transparent ${color}`
+      }
+    }
+  }
+}
+
+const customStyles = theme => ({
+  arrowPopper: arrowGenerator(theme.palette.grey[800]), // TODO should be grey[700] when grey[700] will be supported
+  arrow: {
+    position: 'absolute',
+    fontSize: 6,
+    width: '3em',
+    height: '3em',
+    '&::before': {
+      content: '""',
+      margin: 'auto',
+      display: 'block',
+      width: 0,
+      height: 0,
+      borderStyle: 'solid'
+    }
+  }
+})
```

Comment:
Is there a particular reason why this is not set in the theme ?

Reply:
la class arrow n'existe pas par défaut, tu préfèrerais qu'on ajoute une class spécifique (`className="some-class"`) et déplacer le css dans le theme ?

Reply:
Ah yes OK, I had not seen that the arrow was not supported by default in material UI.

Maybe we can add a name here then ? So that the tooltip can be themed later on.

https://github.com/mui-org/material-ui/blob/next/packages/material-ui/src/Alert/Alert.js#L297

withStyles(customStyles, { name: 'CozyTooltip' })